Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM java:8
+COPY target/payment-to-order-processor.jar /payment-to-order-processor.jar
+EXPOSE 8080
```

Comment:
The same question i asked Martin before - he has not answered ((. [I just moved `Dockerfile` to root directory from `/docker`](https://github.com/commercetools/commercetools-payment-to-order-processor/pull/12/commits/ff7e77bd2c6a54fe511a6c60c7f1cb65b9d9dec1#diff-caedb58fbcfe08727ba3c5b71a196063L1). But agree - the port is redundant here!
Fixed in 3c2611493a40f10ac9134388d0919fb6076e42b8